config.js

```diff
@@ -534,128 +534,128 @@ exports.updateFromArgv = function() {
     }
 
     if (argv.boardsize) {
-        for (let boardsize of argv.boardsize.split(',')) {
-            if (boardsize == "all") {
+        for (let e of argv.boardsize.split(',')) {
```

Comment:
Not sure if it is an improvement if we already named the variable. I'd say `boardsize` is clearer than `e`.